frappe/public/scss/common/indicator.scss

```diff
@@ -67,111 +51,30 @@
 	margin: 0 0 0 4px;
 }
 
-.indicator.green {
-	@include indicator-color('green');
-}
-
-.indicator-pill.green,
-.indicator-pill-right.green,
-.indicator-pill-round.green {
-	@include indicator-pill-color('green');
-}
-
-.indicator.cyan {
-	@include indicator-color('cyan');
-}
-
-.indicator-pill.cyan,
-.indicator-pill-right.cyan,
-.indicator-pill-round.cyan {
-	@include indicator-pill-color('cyan');
-}
-
-.indicator.blue {
-	@include indicator-color('blue');
-}
-
-.indicator-pill.blue,
-.indicator-pill-right.blue,
-.indicator-pill-round.blue  {
-	@include indicator-pill-color('blue');
-}
-
-.indicator.orange {
-	@include indicator-color('orange');
-}
-
-.indicator-pill.orange,
-.indicator-pill-right.orange
-.indicator-pill-round.orange {
-	@include indicator-pill-color('orange');
-}
-
-.indicator.yellow {
-	@include indicator-color('yellow');
-}
-
-.indicator-pill.yellow,
-.indicator-pill-right.yellow
-.indicator-pill-round.yellow {
-	@include indicator-pill-color('yellow');
-}
-
-.indicator.gray,
-.indicator.grey {
-	@include indicator-color('gray');
-}
-
-.indicator-pill.gray,
-.indicator-pill-right.gray,
-.indicator-pill-round.gray,
-.indicator-pill.grey,
-.indicator-pill-right.grey,
-.indicator-pill-round.grey  {
-	@include indicator-pill-color('light-gray');
-}
-
-.indicator.red {
-	@include indicator-color('red');
-}
-
-.indicator-pill.red,
-.indicator-pill-right.red,
-.indicator-pill-round.red {
-	@include indicator-pill-color('red');
-}
-
-.indicator.pink {
-	@include indicator-color('pink');
-}
-
-.indicator-pill.pink,
-.indicator-pill-right.pink,
-.indicator-pill-round.pink {
-	@include indicator-pill-color('pink');
-}
-
-.indicator-pill.darkgrey,
-.indicator-pill-right.darkgrey,
-.indicator-pill-round.darkgrey {
-	@include indicator-pill-color('gray');
-}
-
-.indicator-pill.purple,
-.indicator-pill-right.purple,
-.indicator-pill-round.purple {
-	@include indicator-pill-color('purple');
-}
+$indicator-colors: green, cyan, blue, orange, yellow, gray, grey, red, pink, darkgrey, purple, light-blue;
+@each $color in $indicator-colors {
+	.indicator.#{"" + $color} {
```

Comment:
```suggestion
	.indicator.#{"" + $color} {
```

why `#{"" + $color}`?🤨
at some place you kept `#{$color}`

Reply:
@surajshetty3416 SCSS compiler throws warning when you try to or use a variable as selector name.

<img width="791" alt="Screenshot 2023-01-20 at 7 25 26 PM" src="https://user-images.githubusercontent.com/13928957/213714305-9eeb9bcb-98a5-4bf4-8857-e4b5f7711c6d.png">